File: Cpp/561_Array_Partition_I.cpp
class Solution {
public:
    int arrayPairSum(vector<int>& nums) {
        int sum = 0;
        sort(nums.begin(), nums.end());
        
        for(int i=0; i<nums.size()/2; ++i){
            sum += min(nums[i*2], nums[i*2+1]);
        }
        
        return sum;
    }
};

File: Kotlin/561_Array_Partition_I.kt
class Solution {
    fun arrayPairSum(nums: IntArray): Int {
        var result = 0
        nums.sort()
        for(i in 0 until nums.size step 2){
            result += minOf(nums[i], nums[i+1])
        }
        
        return result
    }
}